src/lazy-image.component.ts

```diff
@@ -143,21 +152,26 @@ export class ImageComponent implements AfterViewInit, OnInit {
 
             image.addEventListener('load', () => {
                 this.loading = false;
+                this.imageWidth = image.width;
```

Comment:
`determineBackground` is able to return empty string... why to go so far in updateing background -> let's improve if above to:
`if (src !== '' && newBackgroundString !== this.backgroundString) {`

Reply:
Well, if the source is empty, it should become empty.